src/main/kotlin/no/nav/syfo/client/pdl/domain/PdlPersonResponse.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.syfo.client.pdl.domain
+
+import java.util.Locale
+
+data class PdlPersonResponse(
+    val errors: List<PdlError>?,
+    val data: PdlHentPerson?
+)
+
+data class PdlHentPerson(
+    val hentPerson: PdlPerson?
+)
+
+data class PdlPerson(
+    val navn: List<PdlPersonNavn>,
+)
+
+data class PdlPersonNavn(
+    val fornavn: String,
+    val mellomnavn: String?,
+    val etternavn: String
+)
+
+fun PdlHentPerson.fullName(): String? {
+    val nameList = this.hentPerson?.navn
+    if (nameList.isNullOrEmpty()) {
+        return null
+    }
+    nameList.first().let {
+        val fornavn = it.fornavn.lowerCapitalize()
+        val mellomnavn = it.mellomnavn
+        val etternavn = it.etternavn.lowerCapitalize()
+
+        return if (mellomnavn.isNullOrBlank()) {
+            "$fornavn $etternavn"
+        } else {
+            "$fornavn ${mellomnavn.lowerCapitalize()} $etternavn"
+        }
+    }
+}
+
+fun String.lowerCapitalize(): String {
+    return this.lowercase()
+        .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
+}
```

Comment:
Tror vi må gjøre slik for å også håndtere navn med bindestrek, feks John-Martin 😄
https://github.com/navikt/syfooversiktsrv/blob/master/src/main/kotlin/no/nav/syfo/util/StringUtil.kt